### DISCUSSION TITLE
ApiIssues

### DISCUSSION THREAD

**Discussion by BhaskarKommu** [marcominerva/SqlDatabaseVectorSearch#2] [General]
hi sir i am facing some problems , while configure apis . 
i am getting error on this line 
var embeddings = await textEmbeddingGenerationService.GenerateEmbeddingsAsync(paragraphs);
Error : Microsoft.SemanticKernel.HttpOperationException: 'HTTP 404 (404)
please help me regarding this . 
please message me in my whatsapp : +91 9381570624

**Comment by marcominerva** [comment]
Check your configuration, in particular the model name.
